app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -3,6 +3,7 @@
   value ||= nil
   type ||= "text"
   describedby ||= false
+  controls ||= nil
```

Comment:
Should we be consistent with `controls` and `describedby`? (set either to `false` or `nil`)